core/src/main/java/bio/terra/pearl/core/service/export/ExportOptionsParsed.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.pearl.core.service.export;
+
+import bio.terra.pearl.core.service.search.EnrolleeSearchExpression;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter @SuperBuilder
+public class ExportOptionsParsed extends ExportOptions {
+    private final EnrolleeSearchExpression searchExpression;
```

Comment:
is there any reason not to add this to `ExportOptions` and just set the expression null before serializing? IMO I think that would be clearer. or maybe non-serialized export options could be `ExportOptionsUIDto` and this is the normal export options. the `Parsed` would throw me off, I think. also, I think maybe this should be called `filterExpression`  so it's clearer it's related to the `filter` string.

Reply:
if we add it to ExportOptions, Jackson will still throw a fit about not knowing how to serialize it, even if it's null.  
and I like having the type indicate whether or not you're getting the parsed expression, because it makes function signatures clearer.   Good points on filter rename, I'll do that in the follow-on PR

Reply:
And as far as DTO -- our convention is that model names without suffixes are what's stored in the DB, and then suffixes ('parsed', 'dto') indicate classes with other purproses.   So ExportOptionsParsed tells you that that's *not* the version stored in the DB.  That said, I'm 100% open to better suffixes than "Parsed".  Maybe "WithExpression"?

Reply:
what about putting `@JsonIgnore` on the expression field? I like "WithExpression" more than "Parsed", though!